app/assets/javascripts/templates/stories/show.jst.ejs

```diff
@@ -166,7 +166,9 @@
       <div class="overall-task-progress">
         <h4>Task progress</h4>
         <div class="progress-bar">
-          <span class="completion"></span>
+          <span class="completion" style="width: {{ taskCompletionPercentage() }}%;">
```

Comment:
does this automatically know to re-run when a task in the story has been updated?